docs/leader-election.md

```diff
@@ -0,0 +1,12 @@
+Epoch Publisher Scheduling (aka “leader election”)
+
+There is no permanent single leader. Any node can publish the next epoch during its time slice in a ring schedule. Concurrent attempts are resolved by etcd using CAS(compare and set).
```

Comment:
There are plenty of `CAS(compare and set)`. First, the space is missing before the opening parenthesis. And second, you probably mean compare-and-swap, not compare-and-set.